tests/common/libtest.sh

```diff
@@ -357,3 +357,97 @@ assert_status_jq() {
     rpm-ostree status --json > status.json
     assert_status_file_jq status.json "$@"
 }
+
+# builds a new RPM and adds it to the testdir's repo
+# $1 - name
+# $2 - version
+# $3 - release
+# $4+ - optional, treated as directive/value pairs
+build_rpm() {
+    local name=$1; shift
+    local version=$1; shift
+    local release=$1; shift
```

Comment:
Maybe default these to `1.0` and `1` respectively?  Or to say it another way, `version` and `release` are keywords but with defaults?